tests/test_basis.py

```diff
@@ -126,6 +129,16 @@ def test_powers(self):
         np.testing.assert_allclose(values, correct_vals, atol=1e-8)
         np.testing.assert_allclose(derivs, correct_ders, atol=1e-8)
 
+    @pytest.mark.unit
+    def test_chebyshev(self):
+        """Test chebyshev function for Chebyshev polynomial evaluation."""
+        l = np.array([0, 1, 2])
+        r = np.linspace(0, 1, 11)  # rho coordinates
+
+        correct_vals = np.array([np.ones_like(r), 2 * r - 1, 8 * r**2 - 8 * r + 1]).T
+        values = chebyshev(r[:, np.newaxis], l, dr=0)
```

Comment:
Could use the evaluate method of the class here to hit that with testing as well